tools/docker/devtoolset/build_devtoolset.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash -eu
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2016 -> 2021

Reply:
Ignore the new files in tools/docker, for I borrow them from //tensorflow/tensorflow/tools/ci_build/.